internal/output/plain.go

```diff
@@ -299,14 +298,14 @@ func sprintTable(vertical bool, slice []interface{}) (string, error) {
 				return "", err
 			}
 
-			if funk.Contains(field.opts, string(OmitEmpty)) && (stringValue == "" || stringValue == nilText) {
-				continue
-			}
-
 			if firstIteration && !funk.Contains(field.opts, string(SeparateLineOpt)) {
 				headers = append(headers, localizedField(field.l10n))
 			}
 
+			if funk.Contains(field.opts, string(OmitEmpty)) && (stringValue == "" || stringValue == nilText) {
+				stringValue = ""
+			}
```

Comment:
This feels inappropriate. OmitEmpty means "don't print if it's empty". This is doing the exact opposite.

The old implementation is definitely awkward, because if we don't omit the column for each row you're going to run into some weird results. But then further down we also have `shiftColsVal` which allows for columns to span based on the `shiftCols=` param. Point is, this may all be by design, and this change you're making is backward incompatible. So either we need to audit existing use of this code and ensure it is compatible with your change, or we don't make any backward incompatible changes.

Reply:
Note this is for the horizontal table mechanic, I suspect you can just revert this back to what it looked like before this PR?

Reply:
Whoops, that's leftover from my initial implementation. It's been reverted now.